build/crypto.m4

```diff
@@ -365,3 +365,31 @@ AC_DEFUN([TS_CHECK_SESSION_TICKET], [
 
   AC_SUBST(has_tls_session_ticket)
 ])
+
+dnl
+dnl Weird versions of BoringSSL do not have SSL_CTX_set_tmp_dh
```

Comment:
Just FYI, SSL_CTX_set_tmp_dh (and DH_get_2048_256) will be deprecated on OpenSSL 3.

Reply:
I'm going to drop this patch and introduce it separately (with conditions for DH_get_2048_256)